install/mysql/glpi-empty.sql

```diff
@@ -8992,8 +8992,8 @@ CREATE TABLE `glpi_refusedequipments` (
   `name` varchar(255) DEFAULT NULL,
   `itemtype` varchar(100) DEFAULT NULL,
   `entities_id` int unsigned NOT NULL DEFAULT '0',
-  `ip` varchar(255) DEFAULT NULL,
-  `mac` varchar(255) DEFAULT NULL,
+  `ip` longtext,
+  `mac` longtext,
```

Comment:
I guess a `text` field could be enough. I am not sure we want to allow more than 64Kb on each field.

Reply:
agree with that